src/main/java/com/launchdarkly/client/EvaluationError.java

```diff
@@ -0,0 +1,10 @@
+package com.launchdarkly.client;
+
+/**
+ * An error indicating an abnormal result from evaluating a feature
+ */
+class EvaluationError extends Exception {
```

Comment:
My original comment was 2 parts, the 2nd part was not addressed/discussed:
Changing the name to EvaluationException is more consistent with Java naming conventions.